projects/packages/videopress/src/client/block-editor/blocks/video/components/banner/style.scss

```diff
@@ -0,0 +1,29 @@
+.block-banner {
+	display: flex;
+    height: 48px;
+    font-size: 14px;
+    align-self: center;
+    align-items: center;
+    background: #F4F4F4;
```

Comment:
I could not find this one at https://color-studio.blog/ ... IMO is good to use one of the variations provided at the gray level (probably gray-0)